src/main.ts

```diff
@@ -39,3 +40,8 @@ export async function run() {
     core.setFailed(error.message);
   }
 }
+
+function isGhes(): boolean {
```

Comment:
might not be a bad idea to add this to toolkit core at some point.

Reply:
+1

lets finish rolling out the feature flags first before adding to the toolkit. Ran into a snag after Ring 1 and in the process of switching to GITHUB_SERVER_URL instead of GITHUB_URL